### PR TITLE
💯💯 Added Monetdb Handler

### LINKED ISSUE #2729: [New Integration]: Integration with MonetDB
### Is there an existing integration?

- [X] I have searched the existing integrations.

### Use Case

**MonetDB** is an open-source column-oriented relational database management system. It is designed to provide high performance on complex queries against large databases, such as combining tables with hundreds of columns and millions of rows.
It  is used worldwide for _education_, _research_, and _businesses_.

### Motivation

As MonetDB is very fast in executing complex queries and it is open source as well.
It is used in education, research and businesses as Data warehouse.

It can be very nice to integrate with mindsdb's amazing ML platforms.

### Implementation

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by parthiv11**
Fixes #2729 

## Please describe what changes you made, in as much detail as possible
  - Added MonetDB Handler And Tested .
  - you can test and Provide your feedback
  - [Installation](https://www.monetdb.org/documentation-Jan2022/admin-guide/installation/)

mindsdb

### CODE REVIEW

File: mindsdb/integrations/handlers/monetdb_handler/monetdb_handler.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                dict for mongo, etc)
+        Returns:
+            HandlerResponse
+        """
+        need_to_close = self.is_connected is False
+        conn = self.connect()
+        cur=conn.cursor() 
+        try:
+            cur.execute(query)
+            print(query , cur.rowcount,query.strip().upper().startswith('SELECT') )
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/monetdb_handler/monetdb_handler.py:132]: Please remove print

File: mindsdb/integrations/handlers/monetdb_handler/monetdb_handler.py [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    
+    def query(self, query: ASTNode) -> StatusResponse:
+        """Receive query as AST (abstract syntax tree) and act upon it somehow.
+        Args:
+            query (ASTNode): sql query represented as AST. May be any kind
+                of query: SELECT, INTSERT, DELETE, etc
+        Returns:
+            HandlerResponse
+        """
+
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/monetdb_handler/monetdb_handler.py:170]: Remove the empty spaces here

File: mindsdb/integrations/handlers/monetdb_handler/monetdb_handler.py [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+        """ Return list of entities
+        Return list of entities that will be accesible as tables.
+        Returns:
+            HandlerResponse: shoud have same columns as information_schema.tables
+                (https://dev.mysql.com/doc/refman/8.0/en/information-schema-tables-table.html)
+                Column 'TABLE_NAME' is mandatory, other is optional.
+        """
+        self.connect()
+        schema = schema_id(connection=self.connection,schema_name=self.schemaName)
+
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/monetdb_handler/monetdb_handler.py:188]: Remove the empty space here

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

**Comment by parthiv11** [comment]
@ZoranPandovski done

**Pull request merged**